Modules/rat_rescue.py

```diff
@@ -106,6 +98,8 @@ def __init__(self, uuid: UUID,
         self._lang_id = lang_id
         self._status = status
         self._hash = None
+        self.__rescue_id = rescue_uuid if rescue_uuid is not None else uuid4()
```

Comment:
What's with the two underscores? This has special meaning in Python: It invokes name mangling. In short, whenever you refer to `self.__rescue_id` here, Python will replace that with `self._Rescue__rescue_id`. This is usually to prevent conflicts in subclasses, I don't believe we need such a thing here.

Reply:
Isn't it marking it as "private" (as private as something can be in Python) too, therefore making PyCharm (and other IDEs) hinting you towards that fact ?

Reply:
@beepbeat, well it's supposed to be private, just not mangled. The attribute is also not accessed from outside the class, which would fail as it couldn't find the non-mangled name.

Reply:
I have marked it as private as it is internal only, i named it as such specifically to invoke name mangling as i fail to see why we would need to expose this attribute as part of `Rescue`'s public interface